lib/asset_checker.rb

```diff
@@ -0,0 +1,37 @@
+class AssetChecker
+  def self.check_files_for_missing(argv)
+    assets_file = 'app/assets/javascripts/assets.js.erb'
+    translations_file = 'app/assets/javascripts/i18n-strings.js.erb'
+    @asset_strings = load_included_strings(assets_file)
+    @translation_strings = load_included_strings(translations_file)
+    argv.any? { |f| missing_strings?(f) } ? 1 : 0
```

Comment:
I would have the method return true/false rather than 1/0 and have the script handle the exit code translation -- and consider renaming this method for clarity

Reply:
Sure